compiler/arser/include/arser/arser.h

```diff
@@ -57,6 +59,29 @@ template <> inline std::string to_string(const char *value) { return std::string
 
 template <> inline std::string to_string(const bool value) { return value ? "true" : "false"; }
 
+/**
+ * @brief Returns the string with the leading dash removed.
+ *
+ * If there is no dash, it returns as it is.
+ */
+std::string remove_dash(const std::string &str)
+{
+  std::string ret{str};
+  auto pos = ret.find_first_not_of('-');
+  if (pos == std::string::npos)
+    return ret;
+  return ret.substr(pos);
+}
+
+/// @brief Returns the string that created by concatenating the elements of a vector with commas.
```

Comment:
Please use one style `/**` or `///`

Reply:
@seanshpark Oh, I thought `///` is for single line and `/**` for larger line. Should it be fixed?

Reply:
> Oh, I thought /// is for single line and /** for larger line.

Actually we didn't have any strict guide for this but usually we used `/**` style, even for one line.

> Should it be fixed?

I hope so...